### DISCUSSION TITLE
datasets-cli

### DISCUSSION THREAD

**Discussion by vitalyshalumov** [huggingface/datasets#3079] [Q&A]
I'm trying to convert a tfds dataset to a huggingface one.

I've tried:

1.       datasets-cli convert  --tfds_path ~/tensorflow_datasets/mnist/3.0.1/ --datasets_directory ~/.cache/huggingface/datasets/mnist/3.0.1/

2.       datasets-cli convert  --tfds_path ~/tensorflow_datasets/mnist/3.0.1/ --datasets_directory ~/.cache/huggingface/datasets/

and other permutations.
The script appears to be running and finishing without an error but when looking in the  huggingface/datasets/ folder nothing is created.

**Comment by mariosasko** [comment]
Hi! The `datasets-cli convert` script hasn't been updated in a while. Luckily, the MNIST dataset is available on the Hub and can be loaded as follows:
```python
from datasets import load_dataset
ds = load_dataset("mnist")
```

**Reply by vitalyshalumov** [reply]
I was more interested in the cli script.
Thank you nonetheless.
